unity-renderer/Assets/DCLPlugins/ECS7/Systems/PointerInputSystem/ECSPointerInputSystem.cs

```diff
@@ -86,7 +85,7 @@ private static void Update(State state)
                 if (curState[i] != prevState[i])
                 {
                     PointerEventType pointerEventType = curState[i] ? PointerEventType.PetDown : PointerEventType.PetUp;
-                    InputAction inputAction = (InputAction)inputActionEnum[i];
+                    InputAction inputAction = (InputAction)_inputActionEnums[i];
```

Comment:
I think this could be just replaced by `inputAction inputAction = (InputAction)i;` and avoid the `Enum.GetValues` completely
@leanmendoza could you confirm?

Reply:
This should work I will wait for @leanmendoza though